### PR TITLE
MTM-41902: Improvement to lwm2m registration doc

### PR DESCRIPTION
**Pull request by sourajitsaha17**

### CODE REVIEW

**Review by MWindel — APPROVED** [review]

**Review by BeateRixen — DISMISSED** [review]

**Review by chbakar — APPROVED** [review]

File: content/protocol-integration/lwm2m-bundle/register-device.md [hunk 1]
```diff
@@ -132,7 +132,8 @@ If the value is not set, the awake time is determined by the LWM2M client's regi
 <tr>
 <td style="text-align: left">generateBootstrapServerConfig</td>
 <td style="text-align: left">Boolean</td>
-<td style="text-align: left">Toggles if {{< product-c8y-iot >}} generates a server config for the LWM2M bootstrap server and writes that back during bootstrap. Default is false.</td>
+<td style="text-align: left">In order to establish a connection between the LWM2M client and the LWM2M bootstrap server on the bootstrap interface, the client requires a pre-loaded Lwm2m bootstrap server account.
+This property enables {{< product-c8y-iot >}} to generate a bootstrap server configuration for the LWM2M bootstrap server and writes it back to the client during bootstrap. Default is false.</td>
```
Comment (Oezge-Akin) [inline content/protocol-integration/lwm2m-bundle/register-device.md:136]: What info does it generate?

**Review by Oezge-Akin — COMMENTED** [review]
There is also a comment about serverPublicKey in the ticket asking if the property is still needed? Have you checked that?

**Comment by sourajitsaha17** [comment]
> There is also a comment about serverPublicKey in the ticket asking if the property is still needed? Have you checked that? --- Yes, we create the server instance with all the bootstrap params and write it during bootstrap. I'm not sure if any customer actively uses it, but according to the LWM2M specs it can be used.

File: content/protocol-integration/lwm2m-bundle/register-device.md (hunk 1 above)
Comment (sourajitsaha17) [reply content/protocol-integration/lwm2m-bundle/register-device.md:136]: It generates the server config info like uri, security mode etc.

Comment (Oezge-Akin) [reply content/protocol-integration/lwm2m-bundle/register-device.md:136]: Can you add that info as well? e.g it sets security mode with NO_SEC.

Comment (sag-eweingaertner) [reply content/protocol-integration/lwm2m-bundle/register-device.md:136]: I would not add this. Rather link the OMA LWM2M 1.0 specification for further explanation.

**Review by Oezge-Akin — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Review by Oezge-Akin — APPROVED** [review]

**Review by sag-eweingaertner — APPROVED** [review]

**Pull request merged**